acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host.platform.start_with?('debian-8')
+      create_remote_file(host,
+                         "/etc/apt/sources.list.d/jessie-backports.list",
+                         "deb http://httpredir.debian.org/debian jessie-backports main")
+    end
```

Comment:
Does an `apt-get update` need to happen after this? (comparing with https://github.com/puppetlabs/puppetserver/pull/1383/files#diff-b1f26fc14676f1a802179a822fdd9377R204)